core/web/middleware.go

```diff
@@ -21,10 +21,8 @@ import (
 // inside this module. To achieve this, we direct webpack to output all of the compiled assets
 // in this module's folder under the "assets" directory.
 
-// HINT: if you are coming here because of this error:
-// `web/middleware.go:28:12: pattern assets: no matching files found`
-// then you must first run `make operator-ui` in the root dir to fetch the frontend
-//nolint
+// nolint
+//
```

Comment:
Was this nolint comment still required? (and is this extra line added by gofmt?) 
We'd like to clean them to be proper machine readable comments, like: `//nolint:<linter>` (no leading space)

Reply:
I don't know which specific rule this is for though